services/stackexchange/stackexchange-monthlyquestions.service.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+const BaseJsonService = require('../base-json')
+const { metric } = require('../../lib/text-formatters')
+const { floorCount: floorCountColor } = require('../../lib/color-formatters')
+const moment = require('moment')
+const Joi = require('joi')
+
+const tagSchema = Joi.object({
+  total: Joi.number()
+    .min(1)
+    .required(),
+}).required()
+
+module.exports = class StackExchangeMonthlyQuestions extends BaseJsonService {
+  static get category() {
+    return 'other'
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'stackoverflow' }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Stack Exchange monthly questions',
+        namedParams: { stackexchangesite: 'stackoverflow', query: 'momentjs' },
+        staticExample: this.render({
+          stackexchangesite: 'stackoverflow',
+          numValue: 2000,
+        }),
+        keywords: ['stackexchange', 'stackoverflow'],
+      },
+    ]
+  }
+
+  static get route() {
+    return {
+      base: 'stackexchange',
+      pattern: ':stackexchangesite/monthlyquestions/:query',
```

Comment:
I feel like this pattern and the one for total questions should be brought into alignment. However total questions uses just `t`. Probably that was for **tag**, or maybe **total**, but I doubt it.

I guess I'd suggest we make this one `stackexchange/stackexchangesite/qm/:query`, where `qm` means **monthly questions**. Then we can think of `t` as standing for **total questions**.